src/libratbag-private.h

```diff
@@ -569,9 +569,9 @@ static inline void
 ratbag_resolution_set_cap(struct ratbag_resolution *res,
 			  enum ratbag_resolution_capability cap)
 {
-	assert(cap <= RATBAG_RESOLUTION_CAP_SEPARATE_XY_RESOLUTION);
+	assert(cap <= MAX_CAP); // no idea what is happening here..
 
-	res->capabilities |= (1 << cap);
+	res->capabilities |= (1 << cap); // ... and here
```

Comment:
Honestly I just guessed.

Reply:
The assert checks the range of capabilities, so it will fail if you try to set an invalid capability. You will need to make `RATBAG_RESOLUTION_CAP_DISABLE` 2 and adjust `MAX_CAP`.

The line below just sets the capability bit.